src/realm/util/file.cpp

```diff
@@ -198,6 +208,18 @@ void remove_dir_recursive(const std::string& path)
 }
 
 
+bool try_remove_dir_recursive(const std::string& path)
+{
+    try {
+        remove_dir_recursive(path);
+    }
+    catch (const File::NotFound&) {
+        return false;
+    }
+    return true;
+}
```

Comment:
@morten-krogh according to your comment above it seems like there will come another update to this PR. Setting status to "in progress"